Added IGV and NGL MoleculeViewer chapters

HammadTheOne opened the pull request:
This PR is a work in progress to add documentation for the `dash-bio` components IGV and NglMoleculeViewer. The documentation adds examples for each of the params, and a default component example in line with existing ones. 

TODO:

- [x] Add IGV chapter
- [x] Add NGL chapter
- [x] Re-generate sitemap
- [x] Update requirements.txt 
- [x] Add parser - Followed up [**Merged and Released to PyPi**](https://github.com/plotly/dash-bio-utils/tree/0.0.6-release)
- [x] Make patch release of 0.6.1 of `dash-bio` with ref fix for NGL [**Merged and Released to PyPi**](https://github.com/plotly/dash-bio/pull/544) 


URLs to visit:

/dash-bio/igv
/dash-bio/nglmoleculeviewer

Post-merge checklist:

The master branch is auto-deployed to `dash.plotly.com`.
Once you have merged your PR, wait 5-10 minutes and check dash.plotly.com
to verify that your changes have been made.

- [ ] I understand

If this PR documents a new feature of Dash:

- [ ] Comment on the original Dash issue with a link to the new docs.
- [ ] Reply to any community thread(s) asking for this feature.

If this PR includes a new dataset available at a remote URL:
- [ ] I have added this dataset to the `datasets/` folder
- [ ] I have added a mapping between the remote URL and the filename in the
`datasets/` folder into the `find_and_replace` dict in `dash_docs/tools.py`

If this PR adds an image or animated GIF:
- [ ] This image was saved and referenced locally rather than via an external link

If I introduced a new relative link inside `dcc.Markdown`:
- [ ] I considered whether I could replace the `dcc.Markdown` call with `rc.Markdown`, which will replace relative links with `tools.relpath` internally. Otherwise, I used e.g. `<dccLink href=tools.relpath('/layout') children="the first chapter"/>` instead of `[the first chapter](/layout)` (importing `tools` from `dash_docs`), and set `dangerously_allow_html=true` in the `dcc.Markdown` call.

alexcjohnson commented:
@HammadTheOne great example apps! Three things I notice:

The dropdown built into IGV doesn't like the padding we apply in our CSS. Seems like it needs an override. Here's what I see:
![Screen Shot 2021-02-16 at 6 45 17 PM](https://user-images.githubusercontent.com/2678795/108136391-fa780e80-7087-11eb-8ca2-cc26e8b00b05.png)
Then if I disable this padding it looks good:
![Screen Shot 2021-02-16 at 6 46 18 PM](https://user-images.githubusercontent.com/2678795/108136405-0237b300-7088-11eb-84ea-28c5572c34d6.png)

In the NGL viewer, this one may be tougher to figure out how to resolve, but when I first load the example page a lot of the molecules are not drawn. Each one will show up after I start interacting with it (changing the related controls, for example). Or if I click off to a different page and come back they usually all show up. I wonder if there are just too many of them? Or something about async loading?

Last, the "see it in action" links for both IGV and NGL point to restricted apps. I'm not sure we need those at all since we have live examples right here, but if we keep them they should be public.

HammadTheOne commented:
@alexcjohnson Turns out the `resetView` prop was the reason for some of the examples not showing, and I've gone ahead and fixed that in e13b9ea. To fix the awkward CSS, I've added an override for that particular selector to `dashbio.css` in the assets directory with 5408d1a . 

For the `See it in action` links, I have yet to deploy those demo apps to the gallery, but once that's done, they'll be accessible, but the links point to where they will eventually belong, but I think those can go up even after merging this PR.

alexcjohnson reviewed (approved):
💃 Fantastic! The `resetView` prop may need a little more thorough documentation in order to help users with this kind of issue, but that should be handled in the dash-bio repo (also when you have nested props like that, it's better if the docstrings are also nested, rather than one big docstring at the top level. Not urgent, but perhaps you can make an issue to follow up on this.)
